arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -818,103 +818,103 @@
+				<string>Show a callout with the latitude and longitude of user-tapped points.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
 				<string>Show legend</string>
 				<key>storyboardName</key>
 				<string>MILShowLegend</string>
 				<key>descriptionText</key>
-				<string>Tap on the button in the toolbar to display legends for the layers on the map</string>
+				<string>Show a legend for all the operational layers in the map.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
 				<string>Simple marker symbol</string>
 				<key>storyboardName</key>
 				<string>SimpleMarkerSymbol</string>
 				<key>descriptionText</key>
-				<string>This sample shows how to create a graphic with simple marker symbol</string>
+				<string>Show a simple marker symbol on a map.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
 				<string>Picture marker symbols</string>
 				<key>storyboardName</key>
 				<string>PictureMarkerSymbols</string>
 				<key>descriptionText</key>
-				<string>This sample shows how to create picture marker symbols from a remote image and an image in assets</string>
+				<string>Use pictures for markers.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
 				<string>Simple renderer</string>
 				<key>storyboardName</key>
 				<string>SimpleRenderer</string>
 				<key>descriptionText</key>
-				<string>This sample demonstrates how to create a simple renderer and add it to a graphics overlay</string>
+				<string>Display common symbols for all graphics in a graphics overlay with a renderer.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
 				<string>Unique value renderer</string>
 				<key>storyboardName</key>
 				<string>UniqueValueRenderer</string>
 				<key>descriptionText</key>
-				<string>This sample demonstrate how to use a unique value renderer to style different features in a feature layer with different symbols</string>
+				<string>Render features in a layer using a distinct symbol for each unique attribute value.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
 				<string>Display grid</string>
 				<key>storyboardName</key>
 				<string>DisplayGrid</string>
 				<key>descriptionText</key>
-				<string>This sample demonstrates how to display and work with coordinate system grids such as LatitudeLongitude, MGRS, UTM and USNG on a map view</string>
+				<string>Display coordinate system grids including Latitude/Longitude, MGRS, UTM and USNG on a map view.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
 				<string>Graphics overlay (dictionary renderer) 3D</string>
 				<key>storyboardName</key>
 				<string>GraphicsOverlayDictionaryRenderer3D</string>
 				<key>descriptionText</key>
-				<string>Demonstrates how to display billboarded point graphics with military symbols.</string>
+				<string>Create graphics using a local style file and an XML file with key/value pairs for each graphic.</string>
```

Comment:
This description only slightly differs from the README but I feel it may be important to include the "3D" part of the description.